src/ui/page/share/dataset/steps/dataSelection/dataSelection.tsx

```diff
@@ -7,8 +9,36 @@ import { FilePicker } from '@/ui/component/filePicker/filePicker'
 import { type Item, List } from '@/ui/component/list/list'
 import { Icon } from '@/ui/component/icon/icon'
 import { useFileStore } from '@/ui/store'
-import type { StoreFilesInput } from '@/domain/file/command'
+import type {
+  FileId,
+  ResourceConflictError,
+  ResourceNotFoundError,
+  StoreFilesInput
+} from '@/domain/file/command'
 import './dataSelection.scss'
+import {
+  useDispatchNotification,
+  type DispatchNotificationInput
+} from '@/ui/hook/useDispatchNotification'
+
+type FileError = ResourceConflictError | ResourceNotFoundError
+
+const logFileError = (error: FileError): void => {
+  switch (error._tag) {
+    case 'resource-conflict': {
+      console.error(
+        "Oops... You are trying either to store a file whose id already exists in memory or to store files with the same id... So we can't store these files..."
```

Comment:
I'm a little latle, but from what I see the resource identifiers in conflict are missing in the error definition, which is unfortunate. So the reported log therefore lacks of context, imho.